ci/src/main/scala/org/typelevel/sbt/CrossRootProject.scala

```diff
@@ -96,14 +96,16 @@ final class CrossRootProject private (
 }
 
 object CrossRootProject {
-  def unapply(root: CrossRootProject): Some[(Project, Project, Project, Project)] =
-    Some((root.all, root.jvm, root.js, root.native))
-
-  private[sbt] def apply(): CrossRootProject = new CrossRootProject(
-    Project("root", file(".")),
-    Project("rootJVM", file(".jvm")),
-    Project("rootJS", file(".js")),
-    Project("rootNative", file(".native"))
+  def unapply(rootProject: CrossRootProject): Some[(Project, Project, Project, Project)] =
+    Some((rootProject.all, rootProject.jvm, rootProject.js, rootProject.native))
+
+  def apply(): CrossRootProject = CrossRootProject("root")
```

Comment:
Eh, I'm not so crazy about a default constructor. Since it was never public we can just remove it.